packages/explat-client/src/internal/local-storage.ts

```diff
@@ -1,15 +1,16 @@
 /**
  * LocalStorage polyfill from https://gist.github.com/juliocesar/926500
  * length and key methods were added to allow for implementing removeExpiredExperimentAssignments
+ * Altered to match window.localStorage signature exactly
  * Exported only for testing purposes
  */
-export const polyfilledLocalStorage = {
+export const polyfilledLocalStorage: typeof window.localStorage = {
```

Comment:
You can use the built-in type `Storage`. Just like you can use `Promise` and others.